### ISSUE TITLE
Bug: Some Planetary Conditions not being assigned from story arc

### ISSUE THREAD

**Issue by AaronGullickson** [AaronGullickson/mekhq#25]
For the Devin scenario, I am noticing that only wind speed is being recorded, not blowing sand and EMI.

**Comment by AaronGullickson** [comment]
Blowing sand and EMI also do not show up in the scenario readout in MekHQ, so it may be an issue there.

**Comment by AaronGullickson** [comment]
Blowing sand is also not working on Twycross, although variable wind conditions is working.

**Comment by AaronGullickson** [comment]
The issue is that blowing sand and EMI are now enums not simple booleans. The method `Scenario.generateInstanceFromXML` actually has some code here to read in the boolean and assign things correctly, but it does not appear to be working.

**Comment by AaronGullickson** [comment]
Also, although it reads in the XML as a true/false, it write out the XML as an ordinal which means it won't read it in again properly and using an ordinal here is bad practice because it depends on the ordering being consistent in the future.

**Comment by AaronGullickson** [comment]
I think I figured out the problem. This happened from a save game. It probably would have worked from a direct load of the story arc, but because I saved the game, it got saved as an ordinal and then when it was reloaded, it didn't read in properly. 

It should be an easy fix to write back out as a boolean rather than an ordinal. I will do some testing and post up an issue/PR on the main MekHQ.
